hass_nabucasa/api.py

```diff
@@ -247,28 +264,38 @@ async def _call_raw_api(
     async def _call_cloud_api(
         self,
         *,
-        path: str,
-        method: str = "GET",
+        url: str | None = None,
```

Comment:
I think it makes more sense to pass the action and call `_action_url` here instead of outside the api class and then passing the URL here. I don't think we will have (or should have) cases when a URL that is not from an action.

Reply:
It can not be done when we need to replace parts of the template with variables like for the files API.

Reply:
OK.

Reply:
changed.